tests/test_integration.ts

```diff
@@ -56,7 +47,18 @@ import { editMercurialVaultDepositorySuite } from './suite/editMercurialVaultDep
     const redeemingFeeInBps = 5;
     const uiRedeemableDepositorySupplyCap = 1_000;
 
-    describe('mercurialVaultDepositorySetupSuite', function () {
+    describe('mercurialVaultDepositorySetupSuite', async function () {
+      const mercurialVaultDepository =
+        await MercurialVaultDepository.initialize({
```

Comment:
nit: this could go into a utility function like:

utils.ts -> createCredixLpDepositoryDevnetUSDC

This would help keeping the boilerplate away

Reply:
this is just temporary, much of them has refactored in Greg's original PR

Reply:
I'm a bit confused, I see "MercurialVaultDepository.initialize"  a lot in the different tests, it might be nice to create a utility function to avoid the boilerplate is just what I meant :)

Reply:
Sounds good, but I didn't see it fixed in #288, did I miss it?